src/helpers/figures/charts/runtime/chartjs_dataset.ts

```diff
@@ -346,7 +346,7 @@ function getTrendingLineDataSet(
 
   return {
     type: "line",
-    xAxisID: TREND_LINE_XAXIS_ID,
+    xAxisID: config.type !== "trailingMovingAverage" ? TREND_LINE_XAXIS_ID : "x",
```

Comment:
At multiple place in the code we rely on `dataSet.xAxisID !== TREND_LINE_XAXIS_ID` to know whether the dataset is a trendline or not. This'll break all of those.  (show values, scatter chart, legend, tooltip, ...)